client/components/fastlane/html/src/v6-fastlane.js

```diff
@@ -0,0 +1,69 @@
+async function onLoad() {
+  async function getBrowserSafeClientToken() {
+    const response = await fetch("/paypal-api/auth/browser-safe-client-token", {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+    const { accessToken } = await response.json();
+
+    return accessToken;
+  }
+
+  const clientToken = await getBrowserSafeClientToken();
+
+  const sdkInstance = await window.paypal.createInstance({
+    clientToken,
+    pageType: "product-details",
+    clientMetadataId: crypto.randomUUID(),
+    components: ["fastlane"],
+  });
+
+  const fastlane = await sdkInstance.createFastlane();
+  window.fastlane = fastlane;
+  fastlane.setLocale("en_us");
```

Comment:
Can lines from 25 to the end of the file technically be moved to the fastlane.js file? And maybe we wrap them in a `setupFastlaneSdk()` function?